### ISSUE TITLE
[question] Home Assistant Supervisor

### ISSUE THREAD

**Issue by a5pire** [whiskerz007/proxmox_hassos_install#66]
With the possible depreciation of the supervised Linux install, could you please confirm that your script doesn’t use this method....

**Comment by willsy555** [comment]
Yes, exactly what I wanted to ask. I've setup on VirtualBox, but this this method may be more resource friendly. Don't want to go down this road if it's using the soon to be redundant supervised install.I'm hoping this script uses one of the virutal Home Assistant images? Cheers.

**Comment by whiskerz007** [comment]
This script does not use the supervised method. This script downloads the prepared disk images that HA creates and imports it into Proxmox. As of this moment, this installation is supported by HA team.
